docs/guides/redis/reconfigure-tls/sentinel.md

```diff
@@ -323,28 +318,6 @@ OK
 
 Now we are going to rotate the certificate of sentinel and database. First let's check the current expiration date of the certificate.
```

Comment:
checking part is removed, then why this line: let's check the current expiration date of the certificate?